### DISCUSSION TITLE
Possible to pass app name via command line?

### DISCUSSION THREAD

**Discussion by patlougheed** [architect/architect#1444] [Q&A]
Hi folks,

Is it possible to pass the arc CLI the app name via the command line, environment variable or some other method?

I've got an Architect app that uses an API that's provisioned by a second, non-Architect CloudFormation stack in a different Git repository. This stack is deployed via CI/CD, and I'm working on having the non-Architect stack populate the appropriate Architect environment variables as part of the CD pipeline. As near as I can tell, in order to get/set the right set of environment variables I need to set the app name via app.arc or similar file for the arc CLI to pick it up. In order to avoid having an additional file I need to keep in sync between repos, I'm looking for the command line method.

Short version:
- Non-Architect stack deploys API
- CD pipeline gets stack outputs, including API Gateway URL
- CD pipeline calls `npc arc env --add --env <env> NAME value` <-- requires app.arc or similar for app name currently; alternative?
- Architect stack deploys

Any and all help appreciated, and if this should be a feature request instead I'm happy to add it there!
